src/Entity/ContentMagicTrait.php

```diff
@@ -200,11 +200,7 @@ public function magicImage(): array
             }
         }
 
-        return [
-            'filename' => '',
-            'alt' => '',
-            'path' => '',
-        ];
+        return [];
```

Comment:
can we return null instead?

Reply:
It breaks Vue, if we do. I've added a `@todo` instead. :-)